content/customize-dock.md

```diff
@@ -23,17 +24,17 @@ redirect_from:
   - /articles/move-launcher
 ---
 
-Pop!\_OS and Ubuntu both contain an application dock on the bottom of the screen. You can adjust this preference, move the dock to the bottom of the screen, and more in <u>Settings</u>.
+Pop!\_OS and Ubuntu both contain an application dock on the screen. You can adjust this preference, such as move the dock to a different side the screen, and more in <u>Settings</u>.
```

Comment:
>You can adjust this preference, such as move the dock to a different side the screen, and more in <u>Settings</u>.

"such as" makes this sentence not work, and it's unclear what "this preference" refers to, since the position was removed from the previous sentence.

If I were to rewrite it, I'd go with something like this:

>The location of the dock, as well as other dock settings, can be adjusted in Settings.